pkgs/top-level/all-packages.nix

```diff
@@ -3123,6 +3123,8 @@ in
 
   magic-vlsi = callPackage ../applications/science/electronics/magic-vlsi { };
 
+  mat2 = callPackage ../tools/graphics/mat2 {};
```

Comment:
Graphics seems like a strange choice, this works for a variety of file types right?

Reply:
Yes, it works with many multimedia types. I'll move it to misc.